src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -103,24 +110,39 @@ const Withdrawal = styled(Td)`
         max-width: 14rem;
     }
 `
-// const replaceLineBreak = (str) => {
-//     return str.toString().replace(/\n/g, '</br>')
-// }
+type ExpandListProps = {
+    data?: {
+        method?: string | ReactElement
+        currencies?: string | ReactElement
+        min_max_deposit?: string | ReactElement
+        min_max_withdrawal?: string | ReactElement
+        deposit_time?: string | ReactElement
+        withdrawal_time?: string | ReactElement
+        description?: string | ReactElement
+        name?: string
+        reference?: string
+        locales?: string[]
+        url?: string
+        reference_link?: string | ReactElement
+    }
```

Comment:
We can declare it outside the type and use like this `data?: DataType`
This need to reduce a read complexity